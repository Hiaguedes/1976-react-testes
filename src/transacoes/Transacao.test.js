import React from 'react';
import { render } from '@testing-library/react';
import Transacao from './Transacao';

describe('Componente de transação do extrato', () => {
    test('Snapshot do componente deve sempre ser o mesmo', () => {
        const { container } = render(<Transacao data="01/01/1111" tipo="saque" valor="20"/>)
        expect(container.firstChild).toMatchSnapshot();
    })
})