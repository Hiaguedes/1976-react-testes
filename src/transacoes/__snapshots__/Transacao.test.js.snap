// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Componente de transação do extrato Snapshot do componente deve sempre ser o mesmo 1`] = `
<div
  class="transacao-container"
>
  <p>
    01/01/1111
  </p>
  <p>
    saque
  </p>
  <p>
    R$ 
    20
  </p>
</div>
`;
